src/gfxinit.cpp

```diff
@@ -284,6 +284,9 @@ static bool SwitchNewGRFBlitter()
 #endif
 		{ "8bpp-optimized",  2,  8,  8,  8,  8 },
 		{ "32bpp-optimized", 0,  8, 32,  8, 32 },
+#ifdef WITH_SSE
+		{ "32bpp-sse2-anim", 1,  8, 32,  8, 32 },
+#endif
 		{ "32bpp-anim",      1,  8, 32,  8, 32 },
```

Comment:
Does the order of this list matter? Can it go directly after 32bpp-sse4-anim without the extra WITH_SSE guard?

Reply:
The order does matter, the implementation selects the first matching and available blitter.

At minimum, 32bpp-sse2-anim should appear after 8bpp-optimized, and before 32bpp-anim.

Reply:
Okay, that makes sense.